src/core/server/saved_objects/service/lib/search_dsl/search_dsl.ts

```diff
@@ -66,11 +66,15 @@ export function getSearchDsl(
     throw Boom.notAcceptable('sortOrder requires a sortField');
   }
 
+  if (namespaces?.some((namespace) => namespace.indexOf('*') >= 0)) {
```

Comment:
outdated - we will allow wildcards through unmodified, since they can't be interpreted as such. See https://github.com/elastic/kibana/pull/67644#discussion_r441582494